unicon_backend/migrations/versions/54e4afebdfe4_initialize_tables.py

```diff
@@ -53,6 +53,16 @@ def upgrade() -> None:
         ),
         sa.PrimaryKeyConstraint("id", name=op.f("pk_project")),
     )
+    op.create_table(
```

Comment:
should groups be a separate migration?

no strong opinions about this, just wanted to get your thoughts on whether groups should be considered as a "new feature" as opposed to it belonging to the base feature set.

Reply:
separated into separate migrations for readability - though i personally prefer lesser migrations as it was easier to autogenerate (but 3 migrations are still ok)